tests/src/test/scala/common/Wsk.scala

```diff
@@ -189,10 +127,10 @@ trait ListOrGetFromCollection extends FullyQualifiedNames {
   }
 }
 
-trait DeleteFromCollection extends FullyQualifiedNames {
+trait DeleteFromCollectionCLI extends BaseDeleteFromCollection {
   self: RunWskCmd =>
 
-  protected val noun: String
+  override protected val noun: String
```

Comment:
Does this value have to be overridden if it is never set in this trait?

Reply:
I think we can remove this line here, since it is defined in BaseDeleteFromCollection.